src/notebook/index.css

```diff
@@ -37,67 +37,87 @@
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget {
+.jp-MarkdownCell-content.p-Widget {
+  padding-left: 5.6px;
```

Comment:
A fractional pixel value doesn't make much sense, except for hi-dpi displays, and then only in extreme cirumstances. Any particular reason this is fractional?